src/cloudai/schema/test_template/nemo_launcher/slurm_install_strategy.py

```diff
@@ -293,3 +294,22 @@ def _clone_repository(self, subdir_path: str) -> None:
         result = subprocess.run(checkout_cmd, cwd=repo_path, capture_output=True, text=True)
         if result.returncode != 0:
             raise RuntimeError(f"Failed to checkout commit: {result.stderr}")
+
+    def _install_requirements(self, subdir_path: str) -> None:
+        """
+        Installs the required Python packages from the requirements.txt file in the cloned repository.
+
+        Args:
+            subdir_path (str): Subdirectory path for installation.
+        """
+        repo_path = os.path.join(subdir_path, self.REPOSITORY_NAME)
```

Comment:
_[minor, feel free to ignore]_ we can use `Path` here as well, not only in tests.

Reply:
I see. Are you suggesting using Path instead of str for the type annotation of subdir_path?

Reply:
Not just type annotation, but actually use `Path` objects instead of `str` + `os.path`.

Reply:
Do you mean something like this?
```
repo_path = Path(subdir_path) / self.REPOSITORY_NAME
```

Reply:
Yes. Plus, `subdir_path` can also be a `Path` object.

Reply:
Okay, let's resolve this in a separate PR. This pattern is widely observed in different places, and we need to update it as a whole.